packages/graphql-server/src/resolvers/area/__tests__/resolvers.test.ts

```diff
@@ -0,0 +1,81 @@
+import areaServices from '../services'
+import { checkPermission } from '../../../utils/check-permission'
+import errors from '../../../errors'
+import { queryArea, queryAreas, mutationCreateArea, mutationUpdateArea } from '../resolvers'
+import { createAreaArgs } from '../__mocks__/createArea'
+import { updateAreaArgs } from '../__mocks__/updateArea'
+import { mockContext } from '../../../__mocks__/context'
+
+jest.mock('../services', () => ({
+  getAreaById: jest.fn(() => 'getAreaById'),
+  getAreas: jest.fn(() => 'getAreas'),
+  createArea: jest.fn(() => 'createArea'),
+  updateArea: jest.fn(() => 'updateArea'),
+}))
+jest.mock('../../../errors', () => ({
+  generateAuthenticationError: jest.fn(() => 'authentication error'),
+}))
+jest.mock('../../../logger')
+jest.mock('../../../utils/check-permission', () => ({
+  checkPermission: jest.fn(() => true),
+}))
+
+describe('queryArea', () => {
+  it('should return correctly', () => {
+    ;(checkPermission as any).mockReturnValue(true)
```

Comment:
Just a small point buy the type here is jest.Mock rather than any, I don't mind too much as it's a test